package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "x3dom",
+	"version": "1.7.3",
+	"description": "A framework for integrating and manipulating X3D scenes as HTML5/DOM elements",
+	"license": "MIT OR GPL-3.0",
+	"keywords": [
+		"3d",
+		"x3d",
+		"x3dom",
+		"webgl"
+	],
+	"homepage": "http://x3dom.org",
+	"main": "dist/x3dom.js",
+	"style": "dist/x3dom.css",
```

Comment:
I appreciate the 'dist' folder does not exist in this repo. Would it be possible to move the dist folder from https://github.com/medihack/x3dom-full-dist into this same repo?